.github/ISSUE_TEMPLATE/post-release-branch-creation.md

```diff
@@ -0,0 +1,79 @@
+---
+name: Post Release Branch Creation Tasks
+about: Tasks to perform after the rc.0 is cut and the upcoming release branch is created
+title: Post Release Branch Creation Tasks for v1.x.y-rc.0
+labels: sig/release, area/release-eng
+---
+
+## As a follow up on: <!-- #<issue> typically the rc.0 cut issue (currently the branch is created by krel during that cut process) -->
+
+## Tasks
+
+<!--
+
+Follow the docs here: https://github.com/kubernetes/sig-release/blob/master/release-engineering/handbooks/post-release-branch-creation.md
+
+Help? Ring @release-managers on slack!
+
+-->
+
+- [ ] Create a thread in #release-management: <!-- Paste link to Slack thread -->
+- [ ] Coordinate the next alpha release cut - <!-- mention @release-managers nudging them to perform the propedeutic tasks for the next alpha (e.g. setting up the new OBS project) -->
+- [ ] (optional) Remove jobs for EOL versions, **only** if agreed upon with Release Managers
+<!--
+Branch Managers might not have a context on if it is "safe" to remove the EOL jobs. We try to be firm with the deadlines and stop cutting patches as soon as we reach the EOL date, but e.g. there might be a new patch needed because of some important security fix, in which case only Release Managers will know about that.
+
+There might be a significant time/delay between the release reaching EOL and the new branch being created, leaving those jobs hanging for a while, which has an impact on the project infra costs.
+
+The trigger for removing such jobs should be solely the EOL date but we shouldn't connect getting rid of EOL jobs and the new branch creation. Even if has been like that before, it shouldn't be longterm.
+-->
+- [ ] Update [`milestone_applier` rules](https://github.com/kubernetes/test-infra/blob/master/config/prow/plugins.yaml)
+- [ ] Update [`kubekins-e2e-v2/variants.yaml`](https://github.com/kubernetes/test-infra/blob/master/images/kubekins-e2e-v2/variants.yaml) with the new version config
+- [ ] Rotate configuration of release branch jobs in kubernetes/test-infra for the upcoming release
+- [ ] Run test generation script, configure the new release dashboards and send a PR with both tests and dashboards config
+- [ ] Monitor the new release dashboard with the Release Signal Lead for at least 48 hours - mainly for missing or misconfigured jobs
```

Comment:
Nit: this should probably be the last item because it's long running and other items are not blocked on it.